src/main/scala/com/fulcrumgenomics/commons/collection/BetterBufferedIterator.scala

```diff
@@ -26,11 +26,17 @@ package com.fulcrumgenomics.commons.collection
 
 import com.fulcrumgenomics.commons.CommonsDef._
 
+/** A little trait to allow BetterBufferedIterator to override the headOption method. The latter method
+  * is not found in [[BetterBufferedIterator]] in 2.11.11, but in 2.12.2! */
+sealed trait HeadOption[A] {
```

Comment:
Since I need to use the `override` keyword when `headOption` is found in `BufferedIterator` in 2.12.